src/prometheus-stack/values/unicorn-values.yaml

```diff
@@ -38,22 +38,22 @@ prometheusOperator:
     patch:
       enabled: true
       image:
-        registry: cgr.dev
-        repository: du-uds-defenseunicorns/kube-webhook-certgen-fips
-        tag: 1.12.2
-      registry: cgr.dev
-      repository: du-uds-defenseunicorns/kube-webhook-certgen-fips
-      tag: 1.10.1
+        registry: quay.io
+        repository: rfcurated/ingress-nginx/kube-webhook-certgen
+        tag: 1.5.3-jammy-fips-rfcurated-rfhardened
+      registry: quay.io
+      repository: rfcurated/ingress-nginx/kube-webhook-certgen
+      tag: 1.5.3-jammy-fips-rfcurated-rfhardened
```

Comment:
```suggestion
```

As far as I can tell these other values aren't used (and the unicorn one had a 1.10.1 image that we didn't include with zarf so I think that's further confirmation).